CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/DatabaseRequestHandler.java

```diff
@@ -310,6 +313,24 @@ public Database changeGetDatabase(Args args){
         return change.getDocumentIDs();
     }
 
+    public void copy(Args args) throws CouchbaseLiteException, IOException {
+      String dbName = args.get("dbName");
+      String dbPath = args.get("dbPath");
+
+      Context context = MainActivity.getAppContext();
+      ZipUtils.unzip(getAsset(dbPath), context.getFilesDir());
+      String dbFileName = new File(dbPath).getName();
+      dbFileName = dbFileName.substring(0, dbFileName.lastIndexOf("."));
+
+      DatabaseConfiguration dbConfig = args.get("dbConfig");
+      File file = new File(context.getFilesDir().getAbsolutePath() + "/" + dbFileName);
+      Database.copy(file, dbName, dbConfig);
+    }
+
+    private InputStream getAsset(String name) {
```

Comment:
what's the use of this function?

Reply:
This will help in getting the zip file from Assets which app internally calls, otherwise it fails to get the zip file